### DISCUSSION TITLE
I am using Streamlit app - How I show token usage in the streamlit with result?

### DISCUSSION THREAD

**Discussion by mraguth** [sinaptik-ai/pandas-ai#785] [Q&A] [answered]
Hi I have a great use case and the results are wonderful in querying my dataframe. Is there a way I can show case 'token usage' at the bottom along with my result?

I am using pandasai -1.3 version.

Thank you so much for all your help!

**Comment by gventuri** [comment, marked as answer]
Hi @mraguth thanks a lot for the kind feedback. This should help you: https://docs.pandas-ai.com/en/latest/LLMs/llms/#count-tokens
